src/main/scala/thurloe/notification/NotificationMonitor.scala

```diff
@@ -121,7 +121,12 @@ class NotificationMonitorActor(val pollInterval: FiniteDuration, pollIntervalJit
 
     lookupShouldNotify(notification) flatMap { shouldNotify =>
       if (shouldNotify) {
-        sendGridDAO.sendNotifications(List(toThurloeNotification(notification))).map(_.headOption)
+        sendGridDAO.sendNotifications(List(toThurloeNotification(notification))).map(_.headOption) recoverWith {
+          case e: KeyNotFoundException =>
+            logger.info(s"Unable to send notification due to missing key: ${e.getMessage}")
+            Future.successful(None)
+          case e => throw e
```

Comment:
i figure it is still worth throwing exceptions if they are for some other reason, but could also be convinced that we should ack them and log the exception at the `warn` level